examples/chip-tool/templates/templates.json

```diff
@@ -26,6 +26,10 @@
         {
             "name": "test_cluster",
             "path": "partials/test_cluster.zapt"
+        },
+        {
+            "name": "TestSuiteHelper_WaitForMs",
```

Comment:
Could you find a prefix that does not contain things like helper/support/core/util in the name?